client/lib/posts/actions.js

```diff
@@ -325,7 +327,7 @@ PostActions = {
 			return;
 		}
 
-		changedAttributes = normalizeApiAttributes( changedAttributes );
+		changedAttributes = PostActions.normalizeApiAttributes( changedAttributes );
```

Comment:
Exposing `normalizeApiAttributes` feels especially awkward on the actions object. Do you think we could rewrite the single test using it so that it doesn't depend so much on testing the return value of `normalizeApiAttributes` and instead test the attributes sent through to wpcom.js ?